client/src/main/webjar/program-form-dialog/program-form-dialog.controller.js

```diff
@@ -206,10 +142,17 @@
         function activate() {
 
             programFormDialog.otherProgramArea = false;
-            programFormDialog.programAreas = getProgramAreas();
 
-            /* TODO: handle query error and write tests for it. */
+            programFormDialog.programAreas = ProgramArea.query();
```

Comment:
No, I should add a loading spinner to the mark-up.  I'll create another ticket and do that.